test/sample-test/Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN cd /python/src/github.com/kubeflow/pipelines
 # Install argo.
 RUN echo "install argo"
 RUN mkdir -p /usr/local/bin/
-#RUN ARGO_VERSION=v2.7.5 && curl -sSL -o /usr/local/bin/argo \
+#RUN ARGO_VERSION=v2.11.6 && curl -sSL -o /usr/local/bin/argo \
 RUN ARGO_VERSION=v2.4.3 && curl -sSL -o /usr/local/bin/argo \
```

Comment:
Not sure about this one either.

Reply:
Hmm good question, I guess this one should not be 2.4.3, since we updated to argo 2.7 .

Reply:
This is used to kick of the ML pipeline sample e2e test, since the changes you do is only for the server side I think we should not update this to 2.11.6. But I think we should go with 2.7.7 since that is the version that is used by Kubeflow today.

Reply:
There was a bug from argo that prevented using the argo cli from a container, that's why we were stuck on 2.4.3. I'm not sure about latest status, if the latest version works, we can upgrade too.
/cc @Ark-kun can you provide context about that issue?